packages/connect-examples/native-android-example/app/src/main/java/com/onekey/hardware/hardwareexample/MainActivity.kt

```
@@ -0,0 +1,891 @@
+        }
+        val json = JsonObject().apply {
+            addProperty("name", "evmGetAddress")
+            add("data", dataJson)
+        }
+        try {
+            webview.callHandler("bridgeCommonCall", json.toString()) { value ->
+                hideLoading()
+                Log.d("evmGetAddress result", value)
+                updateResultText("EVM Address: $value")
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)